lib/bolt/config.rb

```diff
@@ -42,6 +42,12 @@ class Config
 
     PUPPETFILE_OPTIONS = %w[proxy forge].freeze
 
+    DEFAULT_CONFIG_PATHS = [
+      Bolt::Util.windows? ? 'C:\\Program Files\\Puppet Labs\\Bolt\\bolt.yaml' : '/etc/puppetlabs/bolt/bolt.yaml',
```

Comment:
Default paths in Windows should be specified similar to how they are in `Bolt::PuppetDB::Config`: https://github.com/puppetlabs/bolt/blob/master/lib/bolt/puppetdb/config.rb#L20-L23

Hardcoding the path to `C:\Program Files` can cause issues for users who redirect the folder or use a different system drive.